client/src/app/pages/Login.js

```diff
@@ -69,6 +67,13 @@ export default function Login() {
     }
   }, [user]);
 
+
+  const handleResendClick = () => {
+    console.log('here');
+    console.log(createdProfileData);
+    <>{verifyEmail(createdProfileData)}</>;
```

Comment:
is there not a way to do this without putting it the html component brackets?

Reply:
Not that I could figure out, React is pretty stupid in that there is no easy way to call functional components as if they are functions. If anyone knows a way to call it normally that would be great.